internal/host/hostcommands/download_boot_artifacts_cmd.go

```diff
@@ -46,6 +48,10 @@ func (c *downloadBootArtifactsCmd) GetSteps(ctx context.Context, host *models.Ho
 	if err != nil {
 		return nil, err
 	}
+	if !c.imageServiceEnabled {
```

Comment:
There are state transitions we just can't do without this. Maybe we should look into passing this info to the state machine and disabling the transition to the reclaiming states if we don't have the image service instead of erroring here?

Reply:
I think maybe checking this [here](https://github.com/openshift/assisted-service/blob/master/internal/controller/controllers/agent_controller.go#L673-L682) would be better so we never get into this state